modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpNextgenClientCodegen.java

```diff
@@ -189,14 +189,35 @@ public OperationsMap postProcessOperationsWithModels(OperationsMap objs, List<Mo
         objs = super.postProcessOperationsWithModels(objs, allModels);
         OperationMap operations = objs.getOperations();
         for (CodegenOperation operation : operations.getOperation()) {
-            if (operation.returnType == null) {
+            // Once we upgrade to java 21 we can use SequencedSet instead
```

Comment:
Thanks for the PR.

What about using LinkedHashSet (which is what we used in other part of this project to maintain the original order)?

Reply:
That should work, I just didn't find that